src/styles/styles.js

```diff
@@ -1030,6 +1028,10 @@ const styles = {
         borderColor: themeColors.sidebar,
     },
 
+    avatarWrapper: {
+        pointerEvents: 'none',
+    },
```

Comment:
I think maybe the initial guidance was wrong because `pointerEvents` should be a prop on the `View` not a style

https://reactnative.dev/docs/view#pointerevents